src/plugins/intel_gpu/src/graph/permute.cpp

```diff
@@ -108,7 +108,8 @@ std::string permute_inst::to_string(permute_node const& node) {
 }
 
 permute_inst::typed_primitive_inst(network& network, permute_node const& node) :
-        parent(network, node, (!node.can_be_optimized() && node.get_output_layout().is_static()) ? true : false) {
+        parent(network, node, node.can_be_optimized() ? false : node.get_output_layout().is_static() ?
+                                                       true : node.get_output_layout().has_upper_bound() ? true : false) {
```

Comment:
How about reducing ternary operators count? Just to make it more readable...
```suggestion
        parent(network, node, node.can_be_optimized() ? false
                                                      : node.get_output_layout().is_static() || node.get_output_layout().has_upper_bound()) {
```
Or
```suggestion
        !node.can_be_optimized() && (node.get_output_layout().is_static() || node.get_output_layout().has_upper_bound()) {
```

Reply:
Fixed as commented, thanks.